README.md

```diff
@@ -58,15 +58,83 @@ For more information, read the [documentation](https://thelounge.github.io/docs/
 
 ## Development setup
 
-To run the app from source, just clone the code and run this in your terminal:
+
+
+1. Fork lounge to your github account from the top right corner.
+ 
+2. Clone from your github account.
+
+    ```
+    git clone https://github.com/<username>/lounge/
+    ```
+
+3. Install dependencies, it takes a while
+
+    ```
+    cd lounge/
+    npm install
+    ``` 
+
+4. Run lounge to generate configuration files
+
+    ```
+    npm start
+    ```
+
+5. Create new branch and switch into it, name should contain lounge issue ticket 
+    number, create one in https://github.com/thelounge/lounge/issues if missing
+
+    ```
+    git checkout -b XXX-feature-name
+    ``` 
```

Comment:
This is too git-specific.
We don't require branch names to have the issue number in them (though maintainers are encouraged to include username+issue number when they commit to the main repo for organization purposes, but that's more common convention than requirement.
Actually, we don't require branches at all. It sure is a good idea if you are planning to open more than one PR on a project, but to each their own really.